cppcache/integration-test/ThinClientListenerWriter.hpp

```diff
@@ -433,35 +443,35 @@ DUNIT_TASK_DEFINITION(CLIENT2, validateListenerWriterEventsWithNBSTrue)
     ASSERT(parentRegCacheListener->getClears() == 1,
            "region.clear() should be called once");
 
-   auto exmpRegPtr = getHelper()->getRegion(myRegNames[2]);
+    auto exmpRegPtr = getHelper()->getRegion(myRegNames[2]);
     // LOGINFO(" Total Entries in ExampleRegion = %d ", exmpRegPtr->size());
-   ASSERT(exmpRegPtr->size() == 0,
-          "Client-2 ExampleRegion.clear() should have called and so "
-          "Exampleregion size is expected to 0 ");
-
-   // Verify entries in Sub-Region.
-   auto subregPtr1 = exmpRegPtr->getSubregion(myRegNames[3]);
-   auto subregPtr2 = exmpRegPtr->getSubregion(myRegNames[4]);
-
-   // LOGINFO(" Total Entries in SubRegion-1 = %d ", subregPtr1->size());
-   // LOGINFO(" Total Entries in SubRegion-2 = %d ", subregPtr2->size());
-   ASSERT(subRegCacheListener->getCreates() == 5,
-          "should be 5 creates for SubRegion-1 ");
-   ASSERT(subRegCacheListener->getClears() == 0,
-          "should be 0 clears for SubRegion-1 ");
-   ASSERT(subregPtr1->size() == 5,
-          "Client-2 SubRegion-1 should contains 5 entries ");
-   ASSERT(subregPtr2->size() == 5,
-          "Client-2 SubRegion-2 should contains 5 entries ");
-
-   // LOGINFO(" SubRegion-1 CREATES:: subRegCacheListener::m_creates = %d ",
-   // subRegCacheListener->getCreates());
-   // LOGINFO(" SubRegion-1 CLEARS:: subRegCacheListener::m_clears = %d ",
-   // subRegCacheListener->getClears());
-
-   LOGINFO(
-       "validateListenerWriterEventsWithNBSTrue :: Event Validation "
-       "Passed....!!");
+    ASSERT(exmpRegPtr->size() == 0,
+           "Client-2 ExampleRegion.clear() should have called and so "
+           "Exampleregion size is expected to 0 ");
+
+    // Verify entries in Sub-Region.
+    auto subregPtr1 = exmpRegPtr->getSubregion(myRegNames[3]);
+    auto subregPtr2 = exmpRegPtr->getSubregion(myRegNames[4]);
+
+    // LOGINFO(" Total Entries in SubRegion-1 = %d ", subregPtr1->size());
```

Comment:
Pls remove commented code here and below

Reply:
These comments are logging statements that get uncommented when there is a problem with this tests. Given this isn't production code and to be deprecated tests that I am not going to touch logging comments.